keps/sig-storage/2589-csi-migration-portworx/README.md

```diff
@@ -24,50 +72,48 @@ This document present as a vendor specific KEP for the parent KEP
 This inherits all the contents from its parent KEP. It will introduce two new feature gates to be 
 used as described in its parent KEP. For all other contents, please refer to the parent KEP.
 
-### New Feature Gates
+## Motivation
+
+Currently the Portworx volume provisioning happens through Portworx in-tree driver. As part of the parent KEP [CSI Migration](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/625-csi-migration), the Portworx in-tree driver needs to be deprecated in order to give to use Portworx CSI driver.
+
+### Goals
+
+ - To migrate Portworx in-tree plugin to CSI
+
+
+### Non-Goals
+
+  - This doesn't target the core in-tree to CSI migration code in k/k.
+
+## Proposal
+
+The in-tree to CSI migration feature is already in place in k/k. We just need to enable vendor specific feature gates for it to work for each vendor. 
```

Comment:
Change "We just need to enable vendor specific feature gates for it to work for each vendor. " to
We just need to enable Portworx specific feature gates for it to work for Portworx driver.